samples/react/live-events/package.json

```diff
@@ -28,6 +28,7 @@
     "@types/react-dom": "18.0.6",
     "@vitejs/plugin-react": "2.1.0",
     "autoprefixer": "10.4.11",
+    "dotenv": "16.0.2",
```

Comment:
Have you used it in the end?

Reply:
Honestly not sure if it's even required to be listed, Vite might be providing it internally